plugins/auth-backend/src/providers/cloudflare-access/provider.ts

```diff
@@ -277,6 +278,7 @@ export class CloudflareAccessAuthProvider implements AuthProviderRouteHandlers {
         claims,
         cfIdentity,
         expiresInSeconds: claims.exp - claims.iat,
+        jwt,
```

Comment:
This is cached for the full duration of the token, which can cause some races towards the end of the expiration. In this case I think perhaps the best solution might be to not cache the token at all, but add the token to the returned object. Both on line 284 and 266.